articles/migrations/guides/extensibility-node8.md

```diff
@@ -1,57 +1,67 @@
+:::
 
 In this document, we:
 
 * Provide recommendations on how you can ensure a smooth migration for your environment
 * Detail the specific modules effected
 
+
+
+
 ## Summary of the migration
 
-The Webtask runtime powering the following Auth0 features utilize Node.js v4:
+The Webtask runtime powering the following Auth0 features utilize Node 4:
 
 * Rules
 * Hooks
 * Custom database connections
 * Custom social connections
 
-Due to the end of long-term support (LTS) for Node.js v4, we will be migrating the Webtask runtime to use Node.js v8.
 
-As part of this migration, the Auth0 development team has performed extensive testing to detect any breaking changes proactively.
+If you do not use any of the extensibility features mentioned above, you are not affected by this migration. **Additionally, your tenant will automatically be upgraded to use the Node 8 runtime on April 30, 2018.** This will ensure that any future extensibility code you author will be running on a secure runtime.
 
-However, there may be behavioral changes as a result of this migration. As such, we have provided a migration switch that allows you to control the migration of your environment to the new Webtask runtime using Node.js v8.
+Due to the end of long-term support (LTS) for Node 4, we will be migrating the Webtask runtime to use Node 8. As part of this migration, the Auth0 development team has performed extensive testing to detect any breaking changes proactively.
 
-## How to enable the Node.js v8 runtime
+However, there may be behavioral changes as a result of this migration. As such, we have provided a migration switch that allows you to control the migration of your environment to the new Webtask runtime using Node 8.
+
+## How to enable the Node 8 runtime
 
 :::warning
-The Extensibility panel, that this section refers to, will be made available to our cloud customers on **April 17th, 2018**.
+The Extensibility panel, that this section refers to, will be made available to our cloud customers on **April 17, 2018**.
 :::
 
-Node 8 can be enabled through the new Extensibility panel on the [Advanced Tenant Settings](${manage_url}/#/tenant/advanced) page of the Dashboard. This panel will be visible once the new Node 8 runtime is made available on 17 April 2018.
+Node 8 can be enabled through the new Extensibility panel on the [Advanced Tenant Settings](${manage_url}/#/tenant/advanced) page of the Dashboard. This panel will be visible once the new Node 8 runtime is made available on April 17, 2018.
 
 ![Runtime toggle](/media/articles/migrations/node-runtime1.png)
 
-![Runtime toggle options](/media/articles/migrations/node-runtime1.png)
+![Runtime toggle options](/media/articles/migrations/node-runtime2.png)
 
 ::: warning
```

Comment:
Given that you talk about the same things in the next section, I don't see the point of keeping this warning panel, it duplicates the same information.

Reply:
Thanks @mpaktiti - I opted to remove `This panel will be visible once the new Node 8 runtime is made available on April 17, 2018.` instead and keep the warning panel to avoid duplication. This was one of the largest points of confusions for customers so I think having it in the warning panel will call it out more explicitly.

Please let me know if you're ok with this change.